examples/artemis/artemis_persistence_tls.yaml

```diff
@@ -9,7 +9,7 @@ spec:
     persistenceEnabled: true
     messageMigration: true
   acceptors:
-  - name: amqp
-    protocols: amqp
-    port: 5672
-    sslEnabled: true
+    - name: amqp
+      protocols: amqp
+      port: 5672
+      sslEnabled: true
```

Comment:
do you remove the newline at end of file on purpose?

Reply:
No, that was an IDE issue.